examples/clusters_example.py

```diff
@@ -0,0 +1,141 @@
+"""
+Example demonstrating how to use the Clusters API.
+
+This example shows how to:
+- Create a new compute cluster
+- List all clusters
+- Get a specific cluster by ID
+- Get cluster nodes
+- Delete a cluster
+"""
+
+import os
+import time
+
+from verda import VerdaClient
+from verda.constants import Actions, Locations
+
+# Get credentials from environment variables
+CLIENT_ID = os.environ.get('VERDA_CLIENT_ID')
+CLIENT_SECRET = os.environ.get('VERDA_CLIENT_SECRET')
+BASE_URL = os.environ.get('VERDA_BASE_URL', 'https://api.verda.com/v1')
+
+# Create client
+verda = VerdaClient(CLIENT_ID, CLIENT_SECRET, base_url=BASE_URL)
+
+
+def create_cluster_example():
+    """Create a new compute cluster."""
+    # Get SSH keys
+    ssh_keys = [key.id for key in verda.ssh_keys.get()]
+
+    # Check if cluster type is available
+    if not verda.clusters.is_available('16B200', Locations.FIN_03):
+        raise ValueError('Cluster type 16B200 is not available in FIN_03')
+
+    # Get available images for cluster type
+    images = verda.clusters.get_cluster_images('16B200')
+    if 'ubuntu-22.04-cuda-12.9-cluster' not in images:
+        raise ValueError('Ubuntu 22.04 CUDA 12.9 cluster image is not supported for 16B200')
+
+    # Create a 16B200 cluster
+    cluster = verda.clusters.create(
+        hostname='my-compute-cluster',
+        cluster_type='16B200',
+        image='ubuntu-22.04-cuda-12.9-cluster',
+        description='Example compute cluster for distributed training',
+        ssh_key_ids=ssh_keys,
+        location=Locations.FIN_03,
+        shared_volume_name='my-shared-volume',
+        shared_volume_size=30000,
+        wait_for_status=None,
+    )
+
+    print(f'Creating cluster: {cluster.id}')
+    print(f'Cluster hostname: {cluster.hostname}')
+    print(f'Cluster status: {cluster.status}')
+    print(f'Cluster cluster_type: {cluster.cluster_type}')
+    print(f'Location: {cluster.location}')
+
+    # Wait for cluster to enter RUNNING status
+    while cluster.status != verda.constants.cluster_status.RUNNING:
+        time.sleep(2)
```

Comment:
Let's not spam public API

```suggestion
        time.sleep(30)
```